.JuliaFormatter.toml

```diff
@@ -0,0 +1,13 @@
+annotate_untyped_fields_with_any = false
+short_to_long_function_def = false
+long_to_short_function_def = true
```

Comment:
I object here. If I write a long form function, I usually have a reason for that and don't want the formatter to override that

Reply:
It is conditional: it respect the margins, and converts only if there a single statement.